lib/update/phase_bootstrap.go

```diff
@@ -247,9 +247,54 @@ func (p *updatePhaseBootstrap) pullSystemUpdates() error {
 		return trace.Wrap(err, "failed to pull system updates: %s", out)
 	}
 	log.Debugf("Pulled system updates: %s.", out)
+	// the low-level pull-updates call above does not pull teleport config
+	// package updates, so do it here: it's easier since we have access to
+	// plan, cluster state, etc.
+	updates, err := p.collectTeleportUpdates()
```

Comment:
Should we instead make the system part (i.e. pulling updates) an API so it stays in one place? This can lead to code fragmentation where we'd have pieces to do one thing spread over multiple places.

Reply:
We definitely should get rid of `system pull-updates` and use similar approach as here (or in "pull" install phase)... but not as a part of this PR...